### DISCUSSION TITLE
PTZ control on Wyze Cam v3  (HomeAssistant)

### DISCUSSION THREAD

**Discussion by RichardPar** [mrlt8/docker-wyze-bridge#723] [Q&A] [answered]
As the headline, is there a way to control the PTZ on the Wyze Cam v3 ?

Regards,
Richard

**Comment by mrlt8** [comment, marked as answer]
I believe the standard web api or MQTT commands should work with home assistant: https://github.com/mrlt8/docker-wyze-bridge#camera-control

**Reply by RichardPar** [reply]
Thank you... 

I have got the HTTP API working, but not MQTT.  I can see the publish, but no actions.
